tests/Command/RunCommandTest.php

```diff
@@ -207,7 +208,172 @@ public function testExecuteSuccessWithInputInResult(): void
         );
     }
 
-    public function testExecuteSuccessWithInputOutputInResult(): void
+    public function testExecuteSuccessWithLocalInputOutputInResult(): void
```

Comment:
bych to připlácl k některýmu z těch předchozích testů - teď podle mě testExecuteSuccessWithInputInResult netestuje nic, co by nebylo tady

Reply:
Zatím nechám tak, zkusím do toho víc hrábnout v rámci https://keboola.atlassian.net/browse/PS-3496  a ten `RunCommandTest` víc zpřehlednit